fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeStmtTest.java

```diff
@@ -55,4 +56,12 @@ public void testProperties() {
         Assert.assertEquals(1, analyzeStmt.getProperties().size());
         Assert.assertEquals("30", analyzeStmt.getProperties().getOrDefault("expire_sec", "2"));
     }
+
+    @Test
+    public void testHistogram() {
+        String sql = "analyze table t0 update histogram on v1,v2 with 256 buckets";
```

Comment:
This statement is so complex, does our user need to use it manually?

Reply:
1. Histogram calculation is a costly operation, and the general database is triggered manually. 
2. This syntax is compatible with mysql's histogram syntax